tests/framework/extensions/nodes/node_status.go

```diff
@@ -29,19 +37,75 @@ func IsNodeReady(client *rancher.Client, ClusterID string) error {
 		for _, node := range nodes.Data {
 			node, err := client.Management.Node.ByID(node.ID)
 			if err != nil {
-				return false, err
+				return false, nil
```

Comment:
why omit the error here?

Reply:
Because since it's polling, we don't want to exit the poll if we can't find the node we want it to keep polling until it is found.

Reply:
will this not loop forever for other tests if a node is not reachable / in an actual error state?

Reply:
If the node is not reachable then the poll will timeout and the test will fail.